crashsimilarity/utils.py

```diff
@@ -55,17 +59,24 @@ def clean(func):
         return traces
 
     @staticmethod
-    def process(stream, take_top_funcs=None):
+    def process(stream, take_top_funcs=None, compress=False):
```

Comment:
We need more date for better embedding. To handle one month of stack traces for our model i decided to compress them and use vocabulary indexes as words.